### PR TITLE
Series attribute to show empty bins as 0.

### PR DESCRIPTION
**Pull request by yha**
ref #2053

### CODE REVIEW

File: src/args.jl [hunk 1]
```diff
@@ -574,6 +575,7 @@ add_aliases(:xerror, :xerr, :xerrorbar)
 add_aliases(:yerror, :yerr, :yerrorbar, :err, :errorbar)
 add_aliases(:quiver, :velocity, :quiver2d, :gradient, :vectorfield)
 add_aliases(:normalize, :norm, :normed, :normalized)
+add_aliases(:show_empty_bins, :showemptybins, :showempty, :show_empty, :show_zeros, :showzeros)
```
Comment (mkborregaard) [inline src/args.jl]: Do we need the `show_zeros` etc aliases? It seems like they might be appropriated for some other uses.

Comment (yha) [reply src/args.jl]: I added these based on @daschw's suggested name for the attribute, but I don't mind removing them, or doing other name-bikesheddin

**Comment by yha** [comment]
> In any case, new examples have to be added at the end of the `_examples` `Dict` for this process to work. Could you update this PR accordingly?

Do you mean it needs to be moved to the end of the `_examples` array? So it can be placed in its desired position only after updating the reference images?

**Comment by daschw** [comment]
> Do you mean it needs to be moved to the end of the _examples array?

Exactly, array - not dict (my bad). The reference images, e.g. in https://github.com/JuliaPlots/PlotReferenceImages.jl/tree/master/Plots/gr/0.25.0 are named by their index in that array. So, if you add a new reference image in between the Plots test suite will compare wrong images and fail.

**Pull request merged**